### ISSUE TITLE
Log getting erased on plugin update

### ISSUE THREAD

**Issue by ishan576** [strangerstudios/paid-memberships-pro#192]
On updating the plugin all the logs get erased as the plugin files get updated and removes all the logs. Why not create/update the logs in a separate folder outside the plugin folder. Lets say in uploads/pmpro-logs? That way we can preserve the logs.

**Comment by strangerstudios** [comment]
Good idea. I actually don't use those log files anymore and instead use the option to send emails on web hook updates. I think we haven't updated the Authorize.net handler to do that though. Are you using Authorize.net?

**Comment by ishan576** [comment]
Yes, sending email is also a good feature but to have a log file to refer makes more handy (for me). Unfortunately I don't use Authorize.net. I am on PayPal Express and Stripe. It would be really great if this feature is added in up-coming updates.

**Comment by strangerstudios** [comment]
I'll see if we can add a feature to change the location of the log file.

**Comment by strangerstudios** [comment]
We'll use this issue to manage general improvements to the IPN/webhook log system. You can now specify an email address to send the log to vs the file, which is my preferred way of doing things and helps in your case.

We should still move the log file into the wp-content folder somewhere to avoid losing it on updates though.

And we should standardize how the log works across gateways.
